RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByIsolation.cc

```diff
@@ -39,6 +39,10 @@ class PFRecoTauDiscriminationByIsolation : public PFTauDiscriminationProducerBas
     calculateWeights_ = pset.exists("ApplyDiscriminationByWeightedECALIsolation") ?
       pset.getParameter<bool>("ApplyDiscriminationByWeightedECALIsolation") : false;
 
+    // RIC: allow to relax the isolation completely beyond a given tau pt
+    minPtForNoIso_ = pset.exists("minTauPtForNoIso") ?
```

Comment:
I think that it should be OK to leave the fillDescription migration for the next PR.
Tau code has O(10) producers inheriting from PFTauDiscriminationProducerBase.
A full PSet will need to be defined with that fillDescription, both in the base class and in this (and likely other as well) derived classes.
So, it's not a matter of a small set of lines of code change.

Don't expect to easily include more parameters like this in the future.